### PR TITLE
Convert krux to ES6

### PR DESCRIPTION
**Pull request by janua**
👏

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/third-party-tags/krux.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// @flow
+import config from 'lib/config';
+import { getCookie } from 'lib/cookies';
+import { local } from 'lib/storage';
+
+const kruxUrl = '//cdn.krxd.net/controltag?confid=JVZiE3vn';
+
+const retrieve = n => {
```
Comment (katebee) [inline static/src/javascripts/projects/commercial/modules/third-party-tags/krux.js]: Could we use type annotations?

```
const retrieve = (n: string) => {

File: tools/es5to6.json [hunk 2]
```diff
@@ -1,9 +1,5 @@
 {
-  "Francis Carr": [
-    "projects/commercial/modules/creatives/fabric-expanding-v1.js",
-    "projects/common/modules/video/events.js",
-    "projects/commercial/modules/third-party-tags/krux.js"
-  ],
+  "Francis Carr": [],
```
Comment (regiskuckaertz) [inline tools/es5to6.json]: 🎉

**Comment by janua** [comment]
@katebee Typed and removed default export

**Comment by janua** [comment]
@katebee 👀

**Review by katebee — APPROVED** [review]
👍 Woohoo 👏

**Comment by TBonnin** [comment]
This has been reverted since it was causing a lot of 404 errors on the website. https://github.com/guardian/frontend/pull/16993

File: static/src/javascripts/projects/commercial/modules/third-party-tags/krux.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+    return local.getRaw(k) || getCookie(`${k}=([^;]*)`) || '';
+};
+
+const getKruxSegments = (): Array<string> =>
+    retrieve('segs') ? retrieve('segs').split(',') : [];
+
+const shouldRun = config.switches.krux;
+
+export { shouldRun, kruxUrl, getKruxSegments };
```
Comment (NataliaLKB) [inline static/src/javascripts/projects/commercial/modules/third-party-tags/krux.js]: The issue is that `kruxUrl` isn't what should be exported. It needs to be exported as `url` like the [original](https://github.com/guardian/frontend/pull/16955/files#diff-a9822ad8d10ea82b93073afa80b8e6c9L24)

**Pull request merged**